### PR TITLE
Updated header files with fixes to cstyle errors

### PR DESCRIPTION
**Pull request by michizhou**
Closes openebs/openebs#2036.

Signed-off-by: Michael Zhou <myz227@nyu.edu>

### CODE REVIEW

**Review by utkarshmani1997 — APPROVED** [review]
LGTM

**Review by vishnuitta — CHANGES_REQUESTED** [review]
all errors are not fixed. Below is cstyle output on latest files:
```
./cstyle.pl src/istgt_scsi.h 
src/istgt_scsi.h: 29: #define followed by space instead of tab

./cstyle.pl src/istgt_sock.h 
src/istgt_sock.h: 29: #define followed by space instead of tab
src/istgt_sock.h: 35: indent by spaces instead of tabs
src/istgt_sock.h: 46: indent by spaces instead of tabs
```

File: src/istgt_sock.h [hunk 1]
```diff
@@ -31,7 +31,8 @@
 #include <stddef.h>
 #include <unistd.h>
 
-int istgt_getaddr(int sock, char *saddr, int slen, char *caddr, int clen, uint32_t *iaddr, uint16_t *iport);
```
Comment (vishnuitta) [inline src/istgt_sock.h:34]: code changes can be like this:
```
--- a/src/istgt_sock.h
+++ b/src/istgt_sock.h
@@ -26,13 +26,13 @@
  */
 
 #ifndef ISTGT_SOCK_H
-#define ISTGT_SOCK_H
+#define        ISTGT_SOCK_H
 
 #include <stddef.h>
 #include <unistd.h>
 
 int istgt_getaddr(int sock, char *saddr, int slen, char *caddr, int clen,
-                  uint32_t *iaddr, uint16_t *iport);
+    uint32_t *iaddr, uint16_t *iport);
 int istgt_listen(const char *ip, int port, int que);
 int istgt_listen_unx(const char *lpath, int que);
 int istgt_connect(const char *host, int port);
@@ -43,7 +43,7 @@ int istgt_set_recvlowat(int s, int nbytes);
 ssize_t istgt_read_socket(int s, void *buf, size_t nbytes, int timeout);
 ssize_t istgt_write_socket(int s, const void *buf, size_t nbytes, int timeout);
 ssize_t istgt_readline_socket(int sock, char *buf, size_t size, char *tmp,
-                        size_t tmpsize, int *tmpidx, int *tmpcnt, int timeout);
+    size_t tmpsize, int *tmpidx, int *tmpcnt, int timeout);
 ssize_t istgt_writeline_socket(int sock, const char *buf, int timeout);
 
 #endif /* ISTGT_SOCK_H */

```

**Comment by michizhou** [comment]
Hi @vishnuitta, I have incorporated the changes you requested as per your review. Let me know if any other changes are needed. Thanks!

**Review by vishnuitta — CHANGES_REQUESTED** [review]
changes are good. There seems to be an extra file .DS_Store. Can you please remove that?

**Comment by michizhou** [comment]
@vishnuitta I just removed the extra .DS_Store file like you asked.

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**